src/control/server/drpc.go

```diff
@@ -45,6 +45,25 @@ func getDrpcClientConnection(sockDir string) *drpc.ClientConnection {
 	return drpc.NewClientConnection(clientSock)
 }
 
+func checkDrpcClientSocketPath(socketPath string) error {
+	if socketPath == "" {
+		return errors.New("Socket path empty")
+	}
+
+	f, err := os.Stat(socketPath)
+	if err != nil {
+		return errors.Errorf("Socket path '%s' could not be accessed: %s",
```

Comment:
FYI: The format directive %q can be used to quote string arguments.

Reply:
Nice. I wasn't aware of that!